packages/amplify_authenticator/lib/amplify_authenticator.dart

```diff
@@ -250,13 +249,12 @@ class _AuthenticatorState extends State<Authenticator> {
             signUpViewModel: SignUpViewModel(_stateMachineBloc),
             confirmSignUpViewModel: ConfirmSignUpViewModel(_stateMachineBloc),
             confirmSignInViewModel: ConfirmSignInViewModel(_stateMachineBloc),
-
             child: InheritedForms(
+                authStrings: widget.authStrings,
```

Comment:
Shouldn't this "authStrings" argument be part of another InheritedWidget?

It could be `InheritedAuthStrings(authStrings: widget.authStrings, child: InheritedForms)`

Reply:
The problem with doing this, from what I'm seeing, is the timing for instantiating the defaultForms vs the rest of the regular widgets.